ui/src/app/preferences/Preferences.tsx

```diff
@@ -45,15 +44,14 @@ export default function Preferences() {
               <Select
                 id="location"
                 name="location"
-                value={theme || Theme.LIGHT}
+                value={theme || Theme.SYSTEM}
                 options={[
                   { value: Theme.LIGHT, label: 'Light' },
                   { value: Theme.DARK, label: 'Dark' },
                   { value: Theme.SYSTEM, label: 'System' }
                 ]}
                 onChange={(e) => {
                   dispatch(themeChanged(e.target.value as Theme));
-                  nightwind.enable(Theme.DARK === (e.target.value as Theme));
```

Comment:
I removed this as I saw there was a `useEffect` in the `App.tsx` file which handled toggling `nightwind` when the `theme` changes:

https://github.com/flipt-io/flipt/blob/a8ffa95d4d78e9caebf573875c2608e393ef7836/ui/src/App.tsx#L145-L151